VERSION

```diff
@@ -1 +1 @@
-1.2.15-dev
+1.2.14-RC2
```

Comment:
Shouldn't go down from 15-dev to 14-RC2. Even if you built 15 by mistake, just go with it. We don't need to release every number in a sequence, but we don't want to go backwards.

Reply:
There was never a 15 tag, it only ever existed in the Version file, so I think we're good there.